src/psyclone/tests/dynamo0p3_cma_test.py

```diff
@@ -611,7 +612,6 @@ def test_cma_asm_cbanded_dofmap_error():
     _, invoke_info = parse(
```

Comment:
Please update the touched tests to use the `dist_mem` fixture.

Reply:
Done.